domains/_acme-challenge.yahya.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Vxxx9",
+    "email": "github.unflawed727@passinbox.com"
+  },
+  "records": {
+    "CNAME": "e328dc4f089d6d4e6ec689ac._acme.deno.dev."
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, dots at the end, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.